tests/generation/test_utils.py

```diff
@@ -3037,7 +3037,7 @@ def test_eos_token_id_int_and_list_greedy_search(self):
         text = """Hello, my dog is cute and"""
         tokens = tokenizer(text, return_tensors="pt")
 
-        model = GPT2LMHeadModel.from_pretrained("hf-internal-testing/tiny-random-gpt2").to(torch_device)
```

Comment:
I feel it's better to keep it is, but change (line above)

```
tokens = tokenizer(text, return_tensors="pt")
```
to
```
tokens = tokenizer(text, return_tensors="pt").to(torch_device)
```

WDYT?

Reply:
TIL that can be done :D 

Changed!